src/components/PreviewComment/index.js

```diff
@@ -36,17 +34,48 @@ const PreviewComment = ({comment, time, image, totalComment, onPress, user, item
   };
   if (!user) return <></>;
   return (
-    <View testID="userDefined" style={styles.containerPreview}>
-      <View style={styles.lineBeforeProfile} />
-      <View style={styles.container(totalComment)}>
-        <TouchableOpacity style={styles.profileTouchable} onPress={openProfile}>
-          <View style={{left: -16}} />
-          <View style={styles.profile}>
+    <View>
+      <View
+        style={{
+          height: 28,
+          backgroundColor: COLORS.almostBlack,
+          borderBottomLeftRadius: 12,
+          borderBottomRightRadius: 12,
+          borderLeftWidth: 1,
+          borderRightWidth: 1,
+          borderBottomWidth: 1,
+          borderColor: COLORS.darkGray
+        }}>
+        {isShortText && (
+          <LinearGradient
+            colors={['#275D8A', '#275D8A']}
+            style={{
+              position: 'absolute',
+              top: 0,
+              left: 0,
+              right: 0,
+              bottom: 0,
+              borderBottomLeftRadius: 12,
+              borderBottomRightRadius: 12
+            }}
+          />
+        )}
+      </View>
+      <TouchableOpacity
+        style={{
+          paddingHorizontal: 12,
+          height: 56,
+          position: 'absolute',
+          width: '100%'
+        }}
+        onPress={onPress}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, consider moving these inline styles into the StyleSheet at the bottom of the file.

```diff
- <TouchableOpacity
-   style={{
-     paddingHorizontal: 12,
-     height: 56,
-     position: 'absolute',
-     width: '100%'
-   }}
-   onPress={onPress}>
+ <TouchableOpacity style={styles.touchable} onPress={onPress}>
```
In your StyleSheet:
```javascript
touchable: {
  paddingHorizontal: 12,
  height: 56,
  position: 'absolute',
  width: '100%'
}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->